be/src/util/metrics.h

```diff
@@ -159,6 +160,35 @@ class CoreLocalCounter : public Metric {
     CoreLocalValue<T> _value;
 };
 
+class HistogramMetric : public Metric {
+public:
+    HistogramMetric() { clear(); }
```

Comment:
No need to clear() in constructor?

Reply:
Yes, the `_stat.clear()` will be called in `HistogramStat` 's constructor. I will remove it.